### PR TITLE
Execution trace sprint 5

### PR DESCRIPTION
**Pull request by luminaxster**
Hi guys,
Your feedback is always appreciated it. =]
it reflects our [wiki](https://github.com/tlatoza/SeeCodeRun/wiki/Trace-Service-API).

### CODE REVIEW

**Comment by Alaboudi1** [comment]
I have fixed #87. I thought this is a good branch to include the fix with.

File: src/searchTab/trace-search.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+            
+            if(searchBox.traceHelper){
+                let variableValues =searchBox.traceHelper.getValues();
+                let query = searchBox.traceHelper.traceQueryManager.getQuery(variableValues, selectedFilter, value);
+                searchBox.updateTable(searchBox, query);
+            }
+        });
+    }
+    
+    updateTable(searchBox, query){
```
Comment (klein2ms) [inline src/searchTab/trace-search.js]: you could use a repeat.for in the view template for this and just bind the collection to the view.  when elements change in the collection the view would add rows to the table.

File: src/traceService/es-tracer.js [hunk 2]
```diff
@@ -37,11 +26,7 @@ export class EsTracer {
     }
     
     traceChanged(event, results, error = ""){
-        if(this.timeOut){
-            window.clearTimeout(this.timeOut);
-        }
-
-        if(!results){
+         if(!results){
           //  console.log("No trace results found");
```
Comment (klein2ms) [inline src/traceService/es-tracer.js]: remove commented out code from checkin

File: src/searchTab/trace-search.js (hunk 1 above)
Comment (luminaxster) [inline src/searchTab/trace-search.js]: Mason, I will add your recommendation as part of a new issue for sprint 6. Is that OK?

**Comment by luminaxster** [comment]
This branch is "begging" for a merge =]

**Pull request merged**